### ISSUE TITLE
Crash at start after closing game and reopening.

### ISSUE THREAD

**Issue by cpeosphoros** [freeorion/freeorion#460]
More details here: http://www.freeorion.org/forum/viewtopic.php?f=28&t=9923.

I was playing, then some keymaps become irresponsive. I closed the game, intending to reopen and getting them responsive again. When trying to reopen, the game crashes, with this freeorion.log:

```
2016-02-02 12:31:48.317538 [debug] Client : Logger initialized
2016-02-02 12:31:48.318039 [debug] Client : v0.4.5+ [build 2016-01-31.3040556] MSVC 2013
2016-02-02 12:31:48.318039 [debug] Client : GL Version String: 4.5.0 NVIDIA 359.00
2016-02-02 12:31:48.318540 [debug] Client : Dependency versions from headers:
2016-02-02 12:31:48.318540 [debug] Client : Boost: 1_58
2016-02-02 12:31:48.318540 [debug] Client : FreeType: 2.5.5
2016-02-02 12:31:48.318540 [debug] Client : PNG: 1.6.17
2016-02-02 12:31:48.318540 [debug] Client : Python: 2.7.9
2016-02-02 12:31:48.318540 [debug] Client : SDL: 2.0.3
2016-02-02 12:31:48.318540 [debug] Client : zlib: 1.2.8
2016-02-02 12:31:48.323547 [error] Client : chmain.cpp:271 : main() caught exception(GG::FlagSpec::UnknownString): Could not find flag corresponding to unknown string
```

Deleting config.xml solved the problem. I've attached the config.xml file that caused the crash to the forum post.

Diffing that with the default config.xml and incrementally editing it, shows me that the problem is on line 59, which reads like this in the faulty file:

```
<exit>MOD_KEY_LCTRL | MOD_KEY_RCTRL | MOD_KEY_LALT | MOD_KEY_RALT+GGK_RETURN</exit>
```

I didn't manually edit the file that caused the problem, that was auto-generated when I changed the keymap, via options screen, for exiting to desktop to Ctrl-Alt-Return, which, BTW, worked as expected.

So, two related bugs:
1 - The options screen generating invalid code in config.xml.
2 - Config.xml not being correctly processed in statup with that keymap combination.

Unmodded build 2016-01-31.3040556

**Comment by Vezzra** [comment]
@geoffthemedio added a quick fix in 262ddac4f5f348b0dadb652614b83a41f1d3d42e to avoid the crash, but the underlying problem hasn't been solved, so this issue remains open.

**Comment by geoffthemedio** [comment]
Should be fixed.
